h2o-py/tests/testdir_misc/pyunit_export_parquet_multi.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import print_function
+from builtins import range
+import sys
+sys.path.insert(1,"../../../")
+import h2o
+from tests import pyunit_utils
+import string
+import os
+import glob
+import random
+from pandas.util.testing import assert_frame_equal
+
+
+def export_parquet_multipart():
+    test_cases = {"smalldata/prostate/prostate.csv",
+                  "smalldata/titanic/titanic_expanded.csv",
+                  "smalldata/testng/airquality_train1.csv",
+                  "smalldata/gbm_test/autoclaims.csv",
+                  "smalldata/demos/item_demand.csv"}
+
+    for case in test_cases:
```

Comment:
Can we print here what file we are testing and why? 

If it fail, I don't know where unless I count and what is the difference between e.g: 

```
"smalldata/prostate/prostate.csv",
                  "smalldata/titanic/titanic_expanded.csv",
```

can we comment the reasoning why you choose those files? Different sizes, number of cols,...

Reply:
[bca2bf8](https://github.com/h2oai/h2o-3/pull/6190/commits/bca2bf83f7b5828b4a9950d36c6cc5827e0fe92b)